exporters/otlp/internal/otlpconfig/options.go

```diff
@@ -89,16 +93,18 @@ type (
 func NewDefaultConfig() Config {
 	c := Config{
 		Traces: SignalConfig{
-			Endpoint:    fmt.Sprintf("%s:%d", otlp.DefaultCollectorHost, otlp.DefaultCollectorPort),
-			URLPath:     DefaultTracesPath,
-			Compression: otlp.NoCompression,
-			Timeout:     DefaultTimeout,
+			Endpoint:      fmt.Sprintf("%s:%d", otlp.DefaultCollectorHost, otlp.DefaultCollectorPort),
+			URLPath:       DefaultTracesPath,
+			Compression:   otlp.NoCompression,
+			Timeout:       DefaultTimeout,
+			HTTPTransport: http.DefaultTransport,
```

Comment:
These should be the separate transport that we build and maintain.  For just the reasons in the comments above it.